discovery-provider/src/api/v1/playlists.py

```diff
@@ -24,3 +26,18 @@ def get(self, playlist_id):
         playlists = list(map(extend_playlist, playlists))
         response = api_helpers.success_response(playlists, 200, False)
         return response
+
+playlist_tracks_response = make_response("playlist_tracks_response", ns, fields.List(fields.Nested(track)))
+
+@ns.route("/<string:playlist_id>/tracks")
+class PlaylistTracks(Resource):
+    @ns.marshal_with(playlist_tracks_response)
+    def get(self, playlist_id):
+        """Fetch tracks within a playlist"""
+        playlist_id = decode_with_abort(playlist_id, ns)
+        args = {"playlist_id": playlist_id, "with_users": 'true'}
+
+        playlist_tracks = get_playlist_tracks(args)
+        tracks = list(map(extend_track, playlist_tracks))
+        response = api_helpers.success_response(tracks, 200, False)
```

Comment:
I made another `success_response` definition in the API helpers file that removes the need to pass 200 and false btw